cli/internal/util/parse_concurrency.go

```diff
@@ -6,14 +6,16 @@ import (
 	"runtime"
 	"strconv"
 	"strings"
+
+	"github.com/spf13/pflag"
 )
 
 var (
 	// alias so we can mock in tests
 	runtimeNumCPU = runtime.NumCPU
 )
 
-func ParseConcurrency(concurrencyRaw string) (int, error) {
+func parseConcurrency(concurrencyRaw string) (int, error) {
```

Comment:
There's usage text in this function with a typo:

`betteen` should be `between`